src/main/java/net/sf/jabref/gui/PreviewPanel.java

```diff
@@ -306,9 +307,19 @@ public void update() {
         );
         String newValue = sb.toString();
 
-        previewPane.setText(newValue);
-        previewPane.revalidate();
-
+        if (SwingUtilities.isEventDispatchThread()) {
```

Comment:
Is this if statement even necessary? What happens if I'm on the event dispatcher thread and invoke 
````
SwingUtilities.invokeAndWait(() -> {
                    previewPane.setText(newValue);
                    previewPane.revalidate();
                });
````